custom_components/iec/const.py

```diff
@@ -33,3 +33,4 @@
 FUTURE_CONSUMPTIONS_DICT_NAME = "future_consumption"
 STATIC_KWH_TARIFF = "kwh_tariff"
 STATIC_BP_NUMBER = "bp_number"
+ELECTRICITY_INVOICE_ID = "1"
```

Comment:
Please rename to `ELECTRIC_INVOICE_DOC_ID`
```suggestion
ELECTRIC_INVOICE_DOC_ID = "1"
```

Reply:
Done 👍🏻